CHANGELOG.md

```diff
@@ -0,0 +1,75 @@
+# CHANGELOG
+
+We have started this changelogs from version 4.0.0. So, changes on previously released versions can be found in tag branches. Please follow the below format to update add changelogs for new tag version.
+
+## <Tag_Version> (Date)
+### Backward-incompatible changes:
```

Comment:
Maybe change to "Breaking changes". Also maybe add a note to the top of the change log that clarifies a breaking change is anything that either changes the input or output of stix-shifter, or a change that breaks the compatibility between a connector and the core stix-shifter functions.

Reply:
updated